src/adagrams.js

```diff
@@ -1,15 +1,109 @@
+import { LETTER_POOL, SCORE_CHART } from './constant';
+import { cloneObject } from './helpers';
+
+//Implement drawLetters with weighted randomness
 export const drawLetters = () => {
   // Implement this method for wave 1
+  const availableLetters = cloneObject(LETTER_POOL);
+  const selectedLetters = [];
+
+  for (let i = 0; i < 10; i++) {
+    const total = Object.values(availableLetters).reduce((summary, count) => summary + count, 0);
```

Comment:
I very functional implementation to summing the values! Only suggestion is to perhaps to change the name of the `summary` variable to something like `sum` to be more descriptive of the data you are holding.